shopBuddy/shopBuddy/ItemDetailViewController.m

```diff
@@ -40,19 +42,19 @@ - (void)viewDidLoad {
             NSLog(@"😫😫😫 Error getting item details: %@", error.localizedDescription);
         }
     }];
-    /*
-    [[APIManager shared] getItemWithBarcode:self.barcode completion:^(NSDictionary *itemDetails, NSError *error) {
+     */
+    self.barcode = @"3614272049529";
```

Comment:
Do not hardcode. Barcode should be a parameter passed to this VC upon instantiation.